src/components/ReviewVote.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { patchReview } from "../utils/api";
+
+export default function ReviewVote({ review }) {
+    const [votes, Setvotes] = useState(0)
+    const [upVote, setUpVote] = useState(false)
+    const [resetUpVote, setResetUpVote] = useState(false)
+    const [downVote, setDownVote] = useState(false)
+    const [resetDownVote, setResetDownVote] = useState(false)
+    const [err, setErr] = useState(null)
+    const { review_id } = useParams()
+
+    const incUpVote = () => {
+        setErr(null)
+        Setvotes((curr) => {
+            return curr + 1
+        })
+        patchReview(review_id, 1).catch((error) => {
+            setErr(error)
+        })
+        setUpVote(true)
+        setResetUpVote(true)
+    }
+
```

Comment:
It's great that you have caught the error but you will want to remove that vote from the user's perspective should an error occur. Otherwise it could be a confusing user experience if there is an error but it appears that the vote has been counted.